src/components/state-lookup.ts

```diff
@@ -33,14 +33,15 @@ interface StateLookupRoom {
     };
 }
 
-interface StateLookupEvent {
+export interface StateLookupEvent {
     // eslint-disable-next-line camelcase
     room_id: string;
     // eslint-disable-next-line camelcase
     state_key: string;
     type: string;
     // eslint-disable-next-line camelcase
     event_id: string;
+    content: unknown;
```

Comment:
Is content guaranteed to be an `Object` or claimed as here `unknown`? That would require some more type checks in my PR.

Reply:
We could probably call it a `Record<string,unknown>`

Reply:
But if that's not guaranteed by the client lib, we'll get errors like the one #155 fixes, where a presumed object may be a string or a number.

Reply:
Content should really never be anything other than a object, if it is, we should reject it before insertion?

Reply:
https://github.com/matrix-org/matrix-appservice-bridge/pull/196/commits/e027f071835160d106616950564bf8d787d3531f